src/lib/constant/errno.ts

```diff
@@ -8,27 +9,27 @@ const errno = {
   metaMaskUserDeniedMessageSignature: 4001,
   metaMaskWalletRequestPermissions: -32002,
   metaMaskUserRejectedAccountAccess: -32603,
-  metaMaskReplacementTransactionUnderpriced: 'replacement transaction underpriced',
-  metaMaskTransactionHasBeenAborted: 'the transaction has been aborted',
+  metaMaskReplacementTransactionUnderpriced: t`replacement transaction underpriced`,
```

Comment:
These texts are used as error codes and do not need to be translated.

Reply:
Some of the errors can be translated to human readable messages, therefore is included in i18n. Though the actual message is left untranslated in the po file, we can add translation in the future to enhance user experience.